app/views/hub/clients/show.html.erb

```diff
@@ -149,11 +149,12 @@
               <% end %>
               <% unless @client.intake.drop_off? %>
                 <% tax_return = @client.tax_returns.find_by(year: TaxReturn.current_tax_year) %>
+                <% benefits = Efile::BenefitsEligibility.new(tax_return: tax_return, dependents: @client.intake.dependents) if tax_return %>
                 <div class="field-display">
                   <span class="form-question"> Outstanding RRC / Claimed RRC: </span>
-                  <span class="label-value"><%= "$#{tax_return&.outstanding_recovery_rebate_credit}" if tax_return&.outstanding_recovery_rebate_credit %></span>
+                  <span class="label-value"><%= "$#{benefits.outstanding_recovery_rebate_credit}" if benefits&.outstanding_recovery_rebate_credit %></span>
```

Comment:
Should we show a 0 if benefits&.outstanding_recovery_rebate_credit/claimed_recovery_rebate_credit == false? Or protect it from the showing the span or "/" at all?